runtime/gc/gctable.c

```diff
@@ -179,6 +179,7 @@ J9MemoryManagerFunctions MemoryManagerFunctions = {
 #endif /* !J9VM_ENV_DATA64 */
 	j9gc_objaccess_storeObjectToInternalVMSlot,
 	j9gc_objaccess_readObjectFromInternalVMSlot,
+	j9gc_objaccess_arrayObjectDataDisplacement,
```

Comment:
since we have already 'obj' early in the name and since there are other APIs that use indexable rather than array like this one
 
j9gc_objaccess_indexableReadU8


I suggest to rename the API to 

j9gc_objaccess_indexableDataDisplacement

Reply:
let's create a separate PR for this

Reply:
this (and other references of it) should be removed
the rebased code already has j9gc_objaccess_indexableDataDisplacement